### PR TITLE
bugfix/PLAYER-3622 Focus is not restored to Multi-audio menu after clicking Close icon

### PR DESCRIPTION
**Pull request by KseniiaSokolovaEpam**
Focus is not saved on the btn when we use NVDA, so we set the focus forcibly.

[Original task](https://jira.corp.ooyala.com/browse/PLAYER-3622)

Unit tests are passed.

### CODE REVIEW

File: js/components/controlBar.js [hunk 1]
```diff
@@ -858,7 +859,9 @@ var ControlBar = React.createClass({
               ariaLabel={CONSTANTS.ARIA_LABELS.MULTI_AUDIO}
               ariaHasPopup={true}
               ariaExpanded={this.props.controller.state.multiAudioOptions.showPopover ? true : null}
-              onClick={this.handleMultiAudioClick}
+              onClick={function(evt) {
```
Comment (aeng7) [inline js/components/controlBar.js]: Is this change necessary? From the looks of the old code, it looks like the event argument should be passed to handleMultiAudioClick.

Comment (KseniiaSokolovaEpam) [reply js/components/controlBar.js]: Oh, yes! You are right! Thank you.

File: js/components/controlBar.js [hunk 2]
```diff
@@ -248,6 +248,7 @@ var ControlBar = React.createClass({
     } else {
       this.props.controller.toggleMultiAudioScreen();
     }
+    this.handleControlBarFocus(evt);
```
Comment (jpfranco) [inline js/components/controlBar.js]: I'm remembering just now that we have a big refactor coming up where the `handleControlBarFocus` function will be removed from this component (you can see this update in the PLAYER-3475 branch).  Could we switch to the approach that we use for the closed captions menu? You can see how this was implemented in `closedCaptionsPanel.js`

File: tests/components/controlBar-test.js [hunk 3]
```diff
@@ -1130,6 +1130,52 @@ describe('ControlBar', function() {
     expect(popoverStateChanged).toBe(true);
   });
 
+  it('should store multiAudio button focus state', function() {
+    var mockController = {
```
[the remark this answers is not in the document]
Comment (KseniiaSokolovaEpam) [reply tests/components/controlBar-test.js]: Was changed for the [PR](https://github.com/ooyala/html5-skin/pull/1082)

File: js/components/controlBar.js (hunk 2 above)
Comment (KseniiaSokolovaEpam) [reply js/components/controlBar.js]: @jpfranco oh,  i am sorry, I did not get your idea about the approach. But anyway we checked CC and the bug is existing and for CC now.

Comment (jpfranco) [reply js/components/controlBar.js]: Oh ok, it seems to be working for me. Does that only happen when using NVDA?

The problem is that this code has changed a lot, so this focus handler is not even in this file anymore. Please take a look at this PR: https://github.com/ooyala/html5-skin/pull/1083. This logic has been moved to the `PreserveKeyboardFocus` component.

Is it possible to update your fix so that it doesn't rely on the `handleControlBarFocus` function?

Comment (aeng7) [reply js/components/controlBar.js]: Note that 1083 has been merged

**Comment by aeng7** [comment]
@KseniiaSokolovaEpam @jpfranco should this PR be updated due to the recent skin changes?

**Comment by KseniiaSokolovaEpam** [comment]
@aeng7 Now function "handleControlBarFocus" does not exists. (I meant that not only last changes are blockers for merge). So I think that would be better to close  the PR and create new one.
